CONTRIBUTING.md

```diff
@@ -16,7 +16,7 @@ Content:
 
 ## Translating the app
 
-You can translate StreetComplete at POEditor. You can add missing translations and improve existing ones.
+You can translate StreetComplete at POEditor. You can add missing translations and improve existing ones. Discuss translations or specific wording at POEditor and not at GitHub.
```

Comment:
```suggestion
You can translate StreetComplete at POEditor. You can add missing translations and improve existing ones. Discuss translations at POEditor.
```

1) specific wording in English can be discussed on GitHub
2) in some cases English version should be changed to make translation easier or more feasible